### ISSUE TITLE
Logback logs are not reported when JInsight is enabled for some apps

### ISSUE THREAD

**Issue by rshivane** [ApptuitAI/JInsight#5]
Enabling JInsight reports the following error in the logs and stops logging application messages

```
SLF4J: Failed to load class "org.slf4j.impl.StaticMDCBinder".
SLF4J: Defaulting to no-operation MDCAdapter implementation.
SLF4J: See http://www.slf4j.org/codes.html#no_static_mdc_binder for further details.
```

**Comment by rshivane** [comment]
Replication:
 * Do not include slf4j in application
 * Include slf4j in tomcat common classes
When application loads the Logger class, it is loaded by sys classloader from jinsight.jar which does not include a binder to any implementation

**Comment by rshivane** [comment]
Added slf4j-jdk14.jar to shaded jar, so that slf4j logs to JDK logger
Fixed in 3323c274b27448aba869031961bf56605c2e70f6
